flink-ecosystem-backend/src/main/java/org/apache/flink/community/FCPackageController.java

```diff
@@ -46,10 +57,10 @@ public String getTeams() throws JsonProcessingException {
 	}
 
 	@RequestMapping(value = "{inputPackage}", method = RequestMethod.GET, produces = "application/json")
-	public String getPackage(@PathVariable("inputPackage") String pack) throws JsonProcessingException {
-		System.out.println("Get package." + pack);
+	public String getPackage(@PathVariable("inputPackage") Integer packId) throws JsonProcessingException {
```

Comment:
The API spec actually wants a string package identifier (not an integer)

Reply:
Currently, http://localhost:2019/api/v1/packages/14656 works, but it should be http://localhost:2019/api/v1/packages/foo

Reply:
I think the proposed api said `Get a package by id`, so changed from string to integer

Reply:
I agree, it's a bit confusing. But as you can see from the schema, `id` is defined as a string: https://github.com/sorahn/flink-ecosystem/blob/master/api-proposal.md#package

Reply:
will change this.